sdf.md

```diff
@@ -1155,6 +1157,85 @@ Product definitions may set optional defaults and constant values for specific u
 
 The qualities of sdfProduct are the same as for sdfThing and are shown in {{sdfthingqual}}.
 
+
+IANA Considerations {#iana}
+===================
+
+Media Type
+-----------
+
+IANA is requested to add the following Media-Type to the "Media Types" registry.
+
+| Name     | Template             | Reference                 |
+| sdf+json | application/sdf+json | RFC XXXX, {{media-type}}  |
+{: align="left"}
+
+// RFC Ed.: please replace RFC XXXX with this RFC number and remove this note.
+
+{: spacing="compact"}
+Type name:
+: application
+
+Subtype name:
+: sdf+json
+
+Required parameters:
+: none
+
+Optional parameters:
+: none
+
+Encoding considerations:
+: binary (JSON is UTF-8-encoded text)
+
+Security considerations:
+: {{seccons}} of RFC XXXX
+
+Interoperability considerations:
+: none
+
+Published specification:
+: {{media-type}} of RFC XXXX
+
+Applications that use this media type:
+: No known applications currently use this media type.
```

Comment:
OneDM tools?

Reply:
I never understood what this template entry is good for.  
How about

Tools for data and interaction modeling in the IoT

?

Reply:
WFM. We'll probably hear back during media type review if we got it wrong.